torch/fx/experimental/symbolic_shapes.py

```diff
@@ -214,6 +214,10 @@ def is_concrete_int(a: Union[int, SymInt]) -> bool:
     if isinstance(a, int):
         return True
 
+    # note: NestedInt is a concerete int but does not have a.node.expr defined.
+    if a.node.is_nested_int():
+        return True
```

Comment:
If I had a symbolic symnode holding nested int as hint, I think that also return True here, which we may not want.